src/utils/il2cpp-utils-methods.cpp

```diff
@@ -98,7 +99,7 @@ namespace il2cpp_utils {
                     return klass->vtable[offset + slot].method;
                 }
             }
-            logger.error("could not find method in slot %i of interface '%s' in class '%s'!", slot, ClassStandardName(declaringClass).c_str(), ClassStandardName(klass).c_str());
+            il2cpp_utils::getLogger().fmtLog<Paper::LogLevel::ERR>("could not find method in slot {} of interface '{}' in class '{}'!", slot, ClassStandardName(declaringClass).c_str(), ClassStandardName(klass).c_str());
```

Comment:
should use `logger`

Reply:
We can also use our formatter for `Il2CppClass const*` instead of calling these class names directly